EpiAware/src/EpiInfModels/EpiData.jl

```diff
@@ -1,4 +1,4 @@
-"""
+@doc raw"
```

Comment:
Is there any use-case for _not_ using `@doc raw"...some docs..."` pattern?

Reply:
I don't think so?